src/components/App.js

```diff
@@ -2,19 +2,106 @@ import '../styles/App.css';
+        },
+        {
+          isbn: '9780735222359',
+          title: 'THE LINCOLN HIGHWAY',
+          description: 'Two friends who escaped from a juvenile work farm take Emmett Watson on an unexpected journey to New York City in 1954.',
+          amazon_link: 'https://www.amazon.com/dp/0735222355?tag=NYTBSREV-20',
+          author: 'Amor Towles',
+          book_image: 'https://storage.googleapis.com/du-prd/books/images/9780735222359.jpg'
+        },
+        {
+          isbn: '9780593356159',
+          title: 'THE MAID',
+          description: 'When a wealthy man is found dead in his room, a maid at the Regency Grand Hotel becomes a lead suspect.',
+          amazon_link: 'https://www.amazon.com/dp/0593356152?tag=NYTBSREV-20',
+          author: 'Nita Prose',
+          book_image: 'https://storage.googleapis.com/du-prd/books/images/9780593356159.jpg'
+        },
+        {
+          isbn: '9781250274618',
+          title: 'THE CHRISTIE AFFAIR',
+          description: 'Miss Nan O’Dea becomes the mistress of Agatha Christie’s husband.',
+          amazon_link: 'https://www.amazon.com/dp/1250274613?tag=NYTBSREV-20',
+          author: 'Nina de Gramont',
+          book_image: 'https://storage.googleapis.com/du-prd/books/images/9781250274618.jpg'
+        },
+        {
+          isbn: '9780525618584',
+          title: 'CITY OF THE DEAD',
+          description: 'The 37th book in the Alex Delaware series. Delaware and Sturgis investigate a double homicide.',
+          amazon_link: 'https://www.amazon.com/dp/0525618589?tag=NYTBSREV-20',
+          author: 'Jonathan Kellerman',
+          book_image: 'https://storage.googleapis.com/du-prd/books/images/9780525618584.jpg'
+        },
+        {
+          isbn: '9780385546027',
+          title: "THE JUDGE'S LIST",
+          description: 'The second book in the Whistler series. Investigator Lacy Stoltz goes after a serial killer and closes in on a sitting judge.',
+          amazon_link: 'https://www.amazon.com/dp/0385546025?tag=NYTBSREV-20',
+          author: 'John Grisham',
+          book_image: 'https://storage.googleapis.com/du-prd/books/images/9780385546027.jpg'
+        },
+        {
+          isbn: '9780316499774',
+          title: 'THE HORSEWOMAN',
+          description: 'As the Paris Olympics draw near, a mother and daughter, who are champion horse riders, compete against each other.',
+          amazon_link: 'https://www.amazon.com/dp/0316499773?tag=NYTBSREV-20',
+          author: 'James Patterson and Mike Lupica',
+          book_image: 'https://storage.googleapis.com/du-prd/books/images/9780316499774.jpg'
+        }
+      ]
     }
   }
 
   render() {
     return (
-      <h1>book club</h1>
+     <section className='App'>
+       <Switch>
+        <Route exact path='/'>
+          <BookContainer allBooks={this.state.books}/>
+        </Route>
+        <Route exact path='/favorites' render={() => {
+          return <Favorites />
+        }} />
```

Comment:
If these do prove to be equivalent ways of accomplishing the same goal, then at some point we should change one or the other for the sake of cohesion.